fastlane/lanes/release.rb

```diff
@@ -14,10 +14,54 @@
   desc 'Executes the initial steps needed during code freeze'
   lane :code_freeze do |options|
     gutenberg_dep_check
-    ios_codefreeze_prechecks(options)
 
-    ios_bump_version_release
-    new_version = get_app_version
+    Fastlane::Helper::GitHelper.checkout_and_pull(DEFAULT_BRANCH)
+
+    ensure_git_status_clean
```

Comment:
What do you think of running the checks in this order:

```ruby
# First, make sure there's nothing in progress in the working copy
ensure_git_status_clean

# Checkout the up to date default branch, the designated starting point for the code freeze
Fastlane::Helper::GitHelper.checkout_and_pull(DEFAULT_BRANCH)

# Make sure that Gutenberg is configured as expected for a successful code freeze
gutenberg_dep_check
```

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-iOS/commit/1bbc521d256b658597b31f5f255aa29ca40b6e9a